barretenberg/cpp/src/barretenberg/commitment_schemes/shplonk/shplonk.hpp

```diff
@@ -268,5 +268,25 @@ template <typename Curve> class ShplonkVerifier_ {
         // Return opening pair (z, 0) and commitment [G]
         return { { z_challenge, Fr(0) }, G_commitment };
     };
+    /**
+     * @brief Computes \f$ \frac{1}{z - r}, \frac{1}{z+r}, \ldots, \frac{1}{z+r^{2^{d-1}}} \f$.
+     *
+     * @param log_circuit_size \f$ d \f$
+     * @param shplonk_eval_challenge \f$ z \f$
+     * @param gemini_eval_challenge_powers \f$ (r , r^2, \ldots, r^{2^{d-1}}) \f$
+     * @return \f[ \left( \frac{1}{z - r}, \frac{1}{z+r}, \ldots, \frac{1}{z+r^{2^{d-1}}} \right) \f]
+     */
+    static std::vector<Fr> compute_inverted_gemini_denominators(const size_t& num_gemini_claims,
```

Comment:
Nit: there's no reason to use `const size_t&` instead of `const size_t` since the amount of data copied is the same.